### PR TITLE
Zitos speedqueen wash laundry p1

### PR DESCRIPTION
**Pull request by sezito2**
# PR: Wash Laundry p1

## Completed Work

- Created new procedure: `p-zitos-speedqueen-wash-laundry-p1.md`
- Procedure's Goal: teach an intermediate user to use a new laundry machine.

## User Scenario

Elena is a new NC State student moving into a dormitory for the first time. She has done her laundry before at her home, but is unfamiliar with the university’s machines and using a shared laundry room. She wants to clean her clothes in the dorm's laundry room.

## Notable Review Requests

- These instruction are meant for people who have washed laundry before. Is there any information that is too basic for this audience? Is there anything missing
  
- Are the included photos helpful for users?
  
- Should I end the instructions with some conclusion?
  
- Is the introduction/staging sufficient for the audience?

### CODE REVIEW

File: zitos/p-zitos-speedqueen-wash-laundry-p1.md [hunk 1]
```diff
@@ -1,46 +1,48 @@
 # Washing Laundry with NC State Laundry Machines
 
-staging
+University Housing offers all residents living on-campus access to free laundry facilities. Please be mindful of other residents when doing your laundry in our common spaces.
+
+Residents must use their own detergents, dryer sheets, etc.
```
Comment (Ademola-Adepoju) [inline zitos/p-zitos-speedqueen-wash-laundry-p1.md:5]: Hey @sezito2 good work!

My comment on this line "**_Residents must use their own detergents, dryer sheets, etc._**" is that: do you think it feels a little disconnected from the main paragraph? I get that it is supposed to be an Alerting move, yeah? Maybe  you want to make it part of the first step, like a caveat to when they're about to start washing.

Or maybe just make it clearer in it's current position by turning it into a "Note(s)" section.

**Review by Ademola-Adepoju — COMMENTED** [review]
Hey @sezito2 good work!

My comment on this line "**_Residents must use their own detergents, dryer sheets, etc._**" is that: do you think it feels a little disconnected from the first paragraph? Maybe  you want to make it part of the first step, like a caveat to when they're about to start washing.

Or maybe just make it clearer in its current position by turning it into a "Note(s)" section. 

Also, so the readers don't get confused, do you want to list all the things they have to use that's theirs, instead of the etc.

File: zitos/p-zitos-speedqueen-wash-laundry-p1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Washing your Clothes
+
+1. Load your dirty clothes into the washing machine.
+   * **Optional**: separate clothes into whites and colors.
+  
+2. Add detergent to the washer.
```
Comment (justinwatsonn) [inline zitos/p-zitos-speedqueen-wash-laundry-p1.md:12]: I like your use of asterisk to indicate notes/considerations

File: zitos/p-zitos-speedqueen-wash-laundry-p1.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+6. Press start.
+  
+Your laundry is now going through the wash cycle. The control panel will display the remaining in the cycle. 
+
+**Tip**: Set a timer on your phone to keep track of your laundry and remove your laundry promptly.
+
+## Drying your Clothes
+
+1. Load wet clothes into the dryer drum.
+   * **Alert**: Check the lint trap to ensure it is clean.
```
Comment (justinwatsonn) [inline zitos/p-zitos-speedqueen-wash-laundry-p1.md:33]: Could the alert possibly be a step, maybe the first?

**Pull request merged**